opengrok-indexer/src/test/java/org/opengrok/indexer/index/IndexCheckTest.java

```diff
@@ -101,12 +106,16 @@ private void testIndex(boolean projectsEnabled, List<String> subFiles, IndexChec
                 null, null);
         Indexer.getInstance().doIndexerExecution(null, null);
 
-        IndexCheck.isOkay(configuration, mode, subFiles);
+        try (IndexCheck indexCheck = new IndexCheck(configuration, subFiles)) {
+            indexCheck.check(mode);
+        }
     }
 
     @Test
     void testIndexVersionNoIndex() throws Exception {
-        IndexCheck.isOkay(configuration, IndexCheck.IndexCheckMode.VERSION, new ArrayList<>());
+        try (IndexCheck indexCheck = new IndexCheck(configuration)) {
+            indexCheck.check(IndexCheck.IndexCheckMode.VERSION);
```

Comment:
Introducing an assert will be better from junit readability perspective.
assertDoesNotThrow(() -> indexCheck.check());

Reply:
Thanks ! At one point I had a TODO line saying "assert needed ?" and this definitely helps.